stubs/cffi/cffi/cffi_opcode.pyi

```diff
@@ -0,0 +1,94 @@
+from _typeshed import Incomplete
+
+from .error import VerificationError as VerificationError
```

Comment:
Is there a reason to publicly re-export this here? (Same question for most of the `from .foo import bar as bar` imports in the following files as well -- stubgen can get a big over-eager in assuming every import is a public re-export)

```suggestion
```

Reply:
No. I just checked all the files and I think all reexports except for in `__init__.py` can be removed.

However: Two reexports cause stubtest errors when removed (`from .commontypes import COMMON_TYPES` in `cparser.pyi` and `from . import __version_verifier_modules__` in `verifier.pyi`).
I can either

1. Keep the reexports for them, even though they aren't needed
2. Turn them into normal imports, which makes pyright complain about non-accessed import, which I would have to silence or
3. Don't import them, which makes stubtest complain, which I could silence in `stubtest_allowlist.txt`

Reply:
Yup. Stubtest is pretty good at recognising when a class or a function comes from another module (and generally won't complain about missing re-exports in those situations), but struggles a little more (for obvious reasons) to distinguish which constants are defined inside modules, and which are being re-exported from other modules. 

Option (3) is best — just allowlist them, and add a comment to explain why (needless re-exports).